cinder/volume/drivers/nexenta/nfs.py

```diff
@@ -172,10 +166,10 @@ def create_volume(self, volume):
         :return: model update dict for volume reference
         """
         volume_path = self._get_volume_path(volume)
-        volume_options = {
-            'compression': self.volume_compression
-        }
-        self.nms.folder.create_with_props(self.root_path,
+        volume_options = {}
+        if self.volume_compression:
```

Comment:
this will never be False

Reply:
seems like no point in changing from what it previously was

Reply:
self.volume_compression is self.configuration.nexenta_dataset_compression. and self.configuration.nexenta_dataset_compression has no default value.